drivers/volume/aws/aws.go

```diff
@@ -369,8 +369,9 @@ func (a *aws) GetBackupStatus(backup *storkapi.ApplicationBackup) ([]*storkapi.A
 		case "completed":
 			vInfo.Status = storkapi.ApplicationBackupStatusSuccessful
 			vInfo.Reason = "Backup successful for volume"
-			vInfo.TotalSize = uint64(*snapshot.VolumeSize)
-			vInfo.ActualSize = uint64(*snapshot.VolumeSize)
+			// converting to bytes
+			vInfo.TotalSize = uint64(*snapshot.VolumeSize) * 1024 * 1024 * 1024
```

Comment:
Use constants from here: https://github.com/docker/go-units/blob/master/size.go#L24

Reply:
Fixed. Use the following libopenstorage/openstorage/pkg/units/units.go